### ISSUE TITLE
`tree-sitter parse` fails with "symbol not found" if npm package has a scope

### ISSUE THREAD

**Issue by fcurts** [tree-sitter/tree-sitter#277]
I'm working on a `tree-sitter-mylang` grammar modeled after `tree-sitter-javascript`. Once I added an external scanner (copied from tree-sitter-javascript), `tree-sitter parse myfile.mylang` started to fail with:

```
dlsym(0x7f9e4240c300, tree_sitter_sitter_mylang): symbol not found
```

After banging my head against this for a long time, I discovered that the above error is related to the name of my npm package, which is `@mycompany/tree-sitter-mylang`. If I change the name to `tree-sitter-mylang`, the error disappears. However, I need a scoped name, and it's not clear to me what I need to do to make this work.

macOS 10.14.2, npm 6.4.1

Disclaimer: C and Node.js newbie.

**Comment by maxbrunsfeld** [comment]
Ah sorry about this. It’s a regression in version 14. I’ll fix this tomorrow.

**Comment by maxbrunsfeld** [comment]
I believe this is fixed in 0.14.4, which will be published shortly.
